tests/test-data/questionnaire_covid_19_serology_results.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
For future reference:  I think some of the more recent test infrastructure allows for dynamic generation of simulated module responses so we don't need to always create new JSON files.   E.g., using `test_questionnaire_response_dao._submit_questionnaire_response` to generate test payloads (used in a lot of the various COPE survey unit tests).  May need to know the code ID of the consent question to use.

Reply:
Sorry, I just went with the way I know for this one.

Reply:
np, just thought it might make adding additional tests down the line easier